### PR TITLE
Update signed-sms and verify-signed-webhook samples

### PR DESCRIPTION
**Pull request by maxkahan**

### CODE REVIEW

File: sms/verify-signed-sms/.env.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FLASK_APP = "app.py"
 FLASK_ENV = "development"
 FLASK_RUN_PORT = 3000
-FLASK_RUN_HOST = "0.0.0.0"
+# FLASK_RUN_HOST = "0.0.0.0" Uncomment to make the server publicly available
 
 VONAGE_API_KEY=
 VONAGE_SIGNATURE_SECRET=
```
Comment (superchilled) [inline sms/verify-signed-sms/.env.dist:7]: Do these three environment variables need values, or do they even need to be in this file? (e.g. `VONAGE_SIGNATURE_SECRET_METHOD=` is already defined in the top-level `.env.dist` file. Although I guess they're purposely left blank for the instruction:

> Copy `.env.dist` to `.env` and fill in your credentials

in the `README`?

File: sms/verify-signed-sms/.env.dist [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FLASK_APP = "app.py"
 FLASK_ENV = "development"
 FLASK_RUN_PORT = 3000
-FLASK_RUN_HOST = "0.0.0.0"
+# FLASK_RUN_HOST = "0.0.0.0" Uncomment to make the server publicly available
```
Comment (superchilled) [inline sms/verify-signed-sms/.env.dist:4]: If the entire line is uncommented, will the comment itself also not be uncommented and raise a `SyntaxError`? Perhaps add a second `#` before the comment itself?

```suggestion
# FLASK_RUN_HOST = "0.0.0.0" # Uncomment to make the server publicly available
```

Comment (superchilled) [reply sms/verify-signed-sms/.env.dist:4]: Although that could also be confusing, I guess..

**Review by superchilled — COMMENTED** [review]
Hey @maxkahan. Looks good overall. Just had a couple of questions about one of the `.env.dist` files.

Note: I'm not familiar with the dependencies in the `requirements.txt` files, so I'm just assuming these are correct.

File: sms/verify-signed-sms/.env.dist (hunk 1 above)
Comment (maxkahan) [reply sms/verify-signed-sms/.env.dist:7]: I've removed this entirely now, if you run with Ngrok as we recommend it's not needed.

Comment (maxkahan) [reply sms/verify-signed-sms/.env.dist:7]: Have changed the file to be `.flaskenv` to allow us to use the top level `.env` values as well as the Flask environment variables.

File: sms/verify-signed-sms/.env.dist (hunk 2 above)
Comment (maxkahan) [reply sms/verify-signed-sms/.env.dist:4]: I've removed this entirely now, if you run with Ngrok as we recommend it's not needed.

**Review by superchilled — APPROVED** [review]

**Pull request merged**